lib/pure/strutils.nim

```diff
@@ -151,21 +151,29 @@ proc isSpaceAscii*(s: string): bool {.noSideEffect, procvar,
   isImpl isSpaceAscii
 
 proc isLowerAscii*(s: string): bool {.noSideEffect, procvar,
-  rtl, extern: "nsuIsLowerAsciiStr".} =
+  rtl, extern: "nsuIsLowerAsciiStr", deprecated: "See issue #8003".} =
```

Comment:
We don't link to github issues in deprecation messages. Maybe use "There is no agreement on what 'isLowerAscii' should mean"

Reply:
> We don't link to github issues in deprecation messages.

Since when?

It's good to provide this context. But I can make it "There is no agreement on what 'isLowerAscii' should mean; see issue #8003."  if necessary.

Reply:
We never did. I don't remember seeing ``#abc`` in any deprecation warning. Or any other warning for that matter.

Reply:
> But I can make it "There is no agreement on what 'isLowerAscii' should mean; see issue #8003." if necessary.

Better use a full URL to this issue then. This way Nim users do not need to know ``#abc`` refers to a github issue number.

Reply:
@Araq fyi: maybe @dom96 have pasted full URL, but GitHub automatically converted it to "#8003"

Reply:
> Better also use a full URL to this issue then.

+1. I've been an advocate of not marrying any open source git-based project to GitHub, Gitlab or anything else for that matter. The full URL will work even if (very slim chance probably), this project is hosted elsewhere.

Reply:
Our docgen should see find these issue references and make them into hyperlinks for the readers ;)